### PR TITLE
Add identify functionality

### PR DESCRIPTION
**Pull request by aorielly**
This will allow the ability to change launch darkly users and custom attributes on the fly.

### CODE REVIEW

**Comment by aorielly** [comment]
[clabot:check]

File: README.md [hunk 1]
```diff
@@ -204,6 +204,20 @@ reported as enabled:
 http://localhost/users/101?features.show-user-email
 ```
 
+---
+
+### Identify a new user
```
Comment (jacobmoretti) [inline README.md]: Please add a link to this in the table of contents (I probably need to rename that to Table of Contents or something...) - https://github.com/TrueCar/react-launch-darkly#docs

File: test/lib/utils.test.js [hunk 2]
```diff
@@ -76,6 +76,10 @@ describe("lib/utils", () => {
 
   });
 
+  describe("identify", () => {
```
Comment (jacobmoretti) [inline test/lib/utils.test.js]: Please add the actual test. I know some of the other functions don't have any but I'd like to not get more tech debt added.

Comment (aorielly) [reply test/lib/utils.test.js]: I was wondering if that was going to be allowed to fly.

**Pull request merged**